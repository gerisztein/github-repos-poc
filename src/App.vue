<template lang="pug">
  #app
    router-view
</template>

<script>
export default {
  name: 'App'
}
</script>

<style lang="stylus">
body
  font-family -apple-system,BlinkMacSystemFont,"Segoe UI",Helvetica,Arial,sans-serif,"Apple Color Emoji","Segoe UI Emoji","Segoe UI Symbol"
  color #2c3e50
  margin 0
  padding 0
  -webkit-font-smoothing antialiased
  -moz-osx-font-smoothing grayscale
</style>
