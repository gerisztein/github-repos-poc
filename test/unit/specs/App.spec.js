import { shallow, createLocalVue } from 'vue-test-utils'
import App from '@/App'

describe('App.vue', () => {
  let wrapper

  beforeEach(() => {
    wrapper = shallow(App, {
    })
  })

  describe('smoke tests', () => {
    it('should render', () => expect(wrapper.exists()).toBe(true))
  })

  describe('render', () => {
    it('matches snapshot', () => {
      expect(wrapper.html()).toMatchSnapshot()
    })
  })

})
