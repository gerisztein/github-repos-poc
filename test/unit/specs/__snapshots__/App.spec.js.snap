// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App.vue render matches snapshot 1`] = `
<div id="app">
    <header class="main__header"></header>
    <div class="container">
        <!---->
        <div class="app">
            <router-view></router-view>
        </div>
    </div>
</div>
`;
